addons/cluster-autoscaler/cluster-autoscaler.yaml

```diff
@@ -13,9 +13,6 @@
 # limitations under the License.
 
 {{ $version := "UNSUPPORTED" }}
-{{ if eq .Cluster.MajorMinorVersion "1.18" }}
-{{ $version = "v1.18.2 "}}
-{{ end }}
```

Comment:
I think we should still keep 1.18 because it was supported in KKP 2.17. For example, if someone upgrades KKP while still having 1.18 clusters, the cluster-autoscaler might stop working, which is not expected. In the ideal scenario, this shouldn't happen, but it's possible.

Reply:
But we cannot easily support 1.18 because tons of APIs aren't available anymore. We should maybe think more about removing 1.18 from KKP 2.17, as it's EOL for quite some time already.